src/components/input-field/input-field.scss

```diff
@@ -40,60 +41,44 @@
     }
 }
 
+.mdc-text-field {
+    &:not(.mdc-text-field--disabled) {
+        .mdc-floating-label {
+            color: $label-color;
+        }
+    }
+    &.mdc-text-field--disabled {
+        .mdc-floating-label {
+            color: $label-color-disabled;
+        }
+    }
+}
+
 .mdc-text-field {
     width: 100%;
     @include mdc-states-hover-opacity(0);
     @include mdc-states-focus-opacity(0, true);
 
     .mdc-floating-label {
-        &:not(.textarea-label) {
-            top: $floating-label-top-value;
-        }
-
         right: pxToRem(
             8
         ); //This is a hack to force the label truncate when container is too little. Otherwise due to position: absolute it won't. Kia
     }
 }
 
-label,
-div {
-    &.mdc-text-field {
-        transition: background-color 0.2s ease;
-
-        &:not(.mdc-text-field--disabled) {
-            // this has to be written this way to be able to override MD styles
-            background-color: $background-color-normal;
-
-            &:hover {
-                background-color: $background-color-hovered;
-            }
-        }
-
-        &.mdc-text-field--focused {
-            background-color: $background-color-focused;
-        }
+.mdc-text-field__formatted_input {
```

Comment:
Something makes this render as a flex-item, which pushes the input off to the side, instead of letting the input sit above the formatted text (and thus handling clicks as expected).

![image](https://user-images.githubusercontent.com/154725/101502724-95aa8480-3971-11eb-8add-8819bcb0b805.png)

Reply:
I tried setting `position:absolute` on this element, expecting to at least have to style it some further, but that just seemed to fix it 😄

Looking further into why this isn't styled correctly by MDC to begin with, it seems like this isn't a class that comes from MDC at all, it seems to be something we created ourselves, which means the name is not well chosen. We should rename it so it doesn't start with `mdc-text-field`…

Reply:
Ah great!

Good point. I renamed some of these arbitrary class names to things like `lime-text-field-xxxx`. So instead of `mdc-xxx` I've used `lime-xxx` to indicate that this is trying to mimic something with the same name from MD (expecting the same look & feel, or bahaviour). But it's rather our own version which for some reasons we custom-made.

Reply:
If that's helpful, sure. Since we're using shadowDOM, we don't actually _have_ to namespace our classes, so only prefix them when it's helpful 🙂 

(Yeah, I know, there are _loads_ of unnecessarily prefixed classnames, that I originally created, but we didn't always have shadowDOM 😄)

Reply:
Well, as long as they make sens and help the reader in perceiving the code easier, I see no reason in not using lon names and prefixes and so forth. Even if they're in the shadows 😁

Reply:
:zap: